Google Travel Time - Use same info for origin/destination sources (#21846)
home-assistant/home-assistant.github.io: source/_integrations/google_travel_time.markdown

{% include integrations/config_flow.md %}

Notes:

> - Origin and Destination can be the address or the GPS coordinates of the location (GPS coordinates has to be separated by a comma). You can also enter an entity id which provides this information in its state, an entity id with latitude and longitude attributes, or zone friendly name (case sensitive).

Reviewer: ```suggestion
- Origin and Destination can be the address or the GPS coordinates of the location (GPS coordinates have to be separated by a comma). You can also enter an entity ID that provides this information in its state, an entity ID with latitude and longitude attributes, or zone friendly name (case sensitive).
```
